### PR TITLE
push code task manage categories

### PR DESCRIPTION
**Pull request by khaccuong2k**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/xxxx)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by khaccuong2k** [comment]
ready

**Comment by khaccuong2k** [comment]
ready

File: resources/views/admin/category/index.blade.php [hunk 1]
```diff
@@ -27,50 +27,63 @@
...
-                    </td>
-                </tr>
-            </tbody>
-        </table>
-    </div>
+	<div class="pd-20">
+		<h4 class="text-blue h4">@lang('lable.category.list')</h4>
+	</div>
+	<div class="pb-20">
+		<table class="table">
```
Comment (quoctd-0607) [inline resources/views/admin/category/index.blade.php]: indent.  fix toàn bộ file này e

File: routes/web.php [hunk 2]
```diff
@@ -98,11 +58,73 @@
...
+    /**
+     * -----------------------------------
+     * Route for admin                   |
+     * -----------------------------------
+     */
+    /**
+     * Route Get Method
+     * ----------------
+     * route for dashboard
+     */
```
Comment (quoctd-0607) [inline routes/web.php]: remove hết mấy cái remark này đi e

**Comment by khaccuong2k** [comment]
ready

**Pull request merged**